hw4/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h1>HTML Elements</h1>
+        <hr class="hd">
+        <nav>
+            <ul class="top">

```

Comment:
Without styles you can try table. But there is one more way how to make them in one line :)

Reply:
done

Reply:
You added styles into html)) Can we make this part without `style="display: inline-block;"` please?

Reply:
I decided to take the simple route)